src/javascript-node-mongo/devcontainer-template.json

```diff
@@ -21,7 +21,7 @@
                 "16-buster",
                 "14-buster"
             ],
-            "default": "16-bullseye"
+            "default": "18-bullseye"
```

Comment:
Can we bump the minor version ( ie `1.1.0`) ?

https://github.com/devcontainers/templates/blob/be0d3cca168a9b70426b1e27a77cda49388ca8d6/src/javascript-node-mongo/devcontainer-template.json#L3